tools/adjust-package-exports.mts

```diff
@@ -1,12 +1,12 @@
 import { readFile, writeFile } from 'node:fs/promises';
-import { PackageJson } from 'type-fest';
 import { resolve } from 'node:path';
 
 (async () => {
   const dir = resolve('dist');
   const pkg = JSON.parse(await readFile(`${dir}/package.json`, 'utf8'));
-  pkg.exports = mapValues(pkg.exports, (exp: Record<PackageJson.ExportCondition, string>) => ({
+  pkg.exports = mapValues(pkg.exports, (exp: Record<string, string>) => ({
     import: exp.import.replace(/\/src\//, '/').replace(/\.ts$/, '.js'),
+    default: exp.default.replace(/\/src\//, '/').replace(/\.ts$/, '.js'),
     require: exp.require.replace(/\/src\//, '/').replace(/\.ts$/, '.cjs'),
```

Comment:
Still need `require` here?  I'm sure you're not wrong, but I'm curious.

Reply:
Only handling API libs here ATM, since the UI servers are still CJS.

Reply:
This whole setup is wonky & hacky. I think there was an attempt to use src for dependent libraries while here in this monorepo, but then change to reference dist/ when actually published.